<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAgentsDialog</class>
 <widget class="QDialog" name="AddAgentsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>296</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>521</width>
    <height>296</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Agregar agentes</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cantidad de agentes</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbNumOfUnits">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo de agente</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbUnit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>361</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Unidad</string>
   </property>
   <widget class="QSpinBox" name="sbInitialEnergy">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Energía inicial</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbController">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Humano</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Computadora</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="sbMaxVitality">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="value">
     <number>200</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>38</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rapidez</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>79</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitalidad maxima</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Alcance visual</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Energía máxima</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbSpeed">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>70</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>57</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Controlador</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbTeam">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ninguno</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Equipo 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Equipo 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Equipo 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Equipo 4</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="sbInitialVitality">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="value">
     <number>200</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbColorList">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Azul</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Verde</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amarillo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Marron</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Morado</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cian</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rojo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Naranja</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Color de unidad</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>68</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitalidad inicial</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>32</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Equipo</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbVisualScope">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="value">
     <number>200</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbMaxEnergy">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="cmbAgentType">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Unidad</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Unidad de tiro</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="gbShootingUnit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>131</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Unidad de tiro</string>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>28</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Poder</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbPowerValue">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rango</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbRangeValue">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>150</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddAgentsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddAgentsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
